docs/source/using/managing-minishift.adoc

```diff
@@ -89,6 +89,11 @@ $ minishift start --iso-url https://github.com/minishift/minishift-centos-iso/re
 ----
 $ minishift start --iso-url file:///<path_to_ISO_image>
 ----
+- In a WIndows machine the path always starts with a drive letter.
```

Comment:
On a Windows machine ...

Also, you say it, but your example does not reflect it.

Reply:
Will update the document with `file://c:/<path_to_ISO_image>`